modules/lambdas/variables.tf

```diff
@@ -132,6 +132,16 @@ variable "orca_recovery_retrieval_type" {
   description = "AWS glacier recovery type to use. One of Bulk, Standard, Express."
 }
 
+variable "orca_sqs_metadata_queue_arn" {
```

Comment:
Shouldn't need the variables `orca_sqs_metadata_queue_arn` and `orca_sqs_metadata_queue_id` definded as they are not used in the `lambdas/main.tf` file but, you will need to add the variable `orca_sqs_metadata_queue_url` since that is needed by the `copy_to_glacier` lambda based on the changes.

Reply:
Had a wrong name here that confused things. `id` is actually the url, and the `arn` will be required for triggering the new lambda off of the queue.